experiments/adding-new-experiments/add-new-subject.sh

```diff
@@ -0,0 +1,170 @@
+# The root directory for the old subject
```

Comment:
add-new-subject.sh does not seem to describe this file well. The purpose of this file seems to be setting up variables for adding new subjects. Perhaps naming it setup-variables.sh would be more appropriate.